sklearn/calibration.py

```diff
@@ -318,6 +319,7 @@ def _get_estimator(self):
 
         return estimator
 
+    @_fit_context(prefer_skip_nested_validation=False)
```

Comment:
we'd need a comment in the code whenever these are False.

Reply:
done